doc/src/base/collections.md

```diff
@@ -235,6 +236,56 @@ Partially implemented by:
   * [`Iterators.Pairs`](@ref)
 
 ## Set-Like Collections
+A [`Set`](@ref) is a collection of elements, just like an array or dictionary, with no duplicated elements.
+
+The two important differences between a set and other types of collection is that in a set you can have only one of each element, and, in a set, the order of elements isn't important (whereas an array can have multiple copies of an element and their order is remembered).
+```julia
+julia> colors = Set(["yellow", "blue", "green", "red"])
+Set(["yellow", "blue", "green", "red"])
+```
+
+You can use [`push!`](@Ref) to add elements to a set:
+```julia
+julia> push!(colors, "black")
+Set(["yellow", "blue", "green", "black", "red"])
+```
+
+If you try to add something to the set that's already there nothing will happen, because sets don't store repeated elements.
+
+You can't use [`pushfirst!`](@Ref), because that works only for things that have a concept of "first", like arrays.
+
+To see if something is in the set, you can use [`in`](@Ref):
+```julia
+julia> in("green", colors)
+true
+```
+or using infix notation
+```julia
+julia> "green" in colors
+true
+```
+
+There are some standard operations you can do with sets, namely find their `union`, `intersection`, and `difference`, with the functions, [`union`](@Ref), [`intersect`](@Ref), and [`setdiff`](@Ref):
```

Comment:
I don't think the first set of `union`, `intersection` and `difference` should be code quoted since it isn't code (there isn't a `difference` function, for example).